src/color-slider/color-slider.css

```diff
@@ -5,7 +5,16 @@
 
 .color-slider,
 .slider-tooltip {
-	--transparency: repeating-conic-gradient(transparent 0 25%, rgb(0 0 0 / .05) 0 50%) 0 0 / 1.5em 1.5em content-box border-box;
+	--transparency-cell-size: 1.5em;
+
+	--_transparency-cell-size: var(--transparency-cell-size, .5em);
+	--_transparency-background: var(--transparency-background, transparent);
+	--_transparency-darkness: var(--transparency-darkness, 5%);
+	--_transparency: var(--transparency-grid,
```

Comment:
Oh I see. The `--_name` variables should match the regular variable names (plus the `_`).

Reply:
```suggestion
	--_transparency-grid: var(--transparency-grid,
```